beetsplug/discogs.py

```diff
@@ -305,7 +311,15 @@ def get_album_info(self, result):
         # Explicitly reload the `Release` fields, as they might not be yet
         # present if the result is from a `discogs_client.search()`.
         if not result.data.get("artists"):
-            result.refresh()
+            try:
+                result.refresh()
+            except CONNECTION_ERRORS:
+                self._log.debug(
+                    "Connection error in release lookup: {0}",
+                    result,
+                    exc_info=True,
```

Comment:
I doubt that the exception trace will be useful to the user - I think the `result` is enough.
```suggestion
```